### PR TITLE
MOB-1413 - Added ability to switch messaging profile

### PR DESCRIPTION
**Pull request by Oleg-Pecheneg**

### CODE REVIEW

File: unstoppable-ios-app/domains-manager-ios/Modules/DomainProfile/Cells/ManageDomainRecordCell/ManageDomainRecordCell.swift [hunk 1]
```diff
@@ -209,12 +209,11 @@ private extension ManageDomainRecordCell {
         case .edit(let callback):
             return UIAction(title: action.title, image: action.icon, identifier: .init(UUID().uuidString), handler: { _ in callback() })
         case .editForAllChains(let chains, let callback):
-            if #available(iOS 15.0, *) {
-                let subtitle = chains.prefix(3).map({ $0 }).joined(separator: ", ")
-                return UIAction(title: action.title, subtitle: subtitle, image: action.icon, identifier: .init(UUID().uuidString), handler: { _ in callback() })
-            } else {
-                return UIAction(title: action.title,  image: action.icon, identifier: .init(UUID().uuidString), handler: { _ in callback() })
-            }
+            let subtitle = chains.prefix(3).map({ $0 }).joined(separator: ", ")
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/Modules/DomainProfile/Cells/ManageDomainRecordCell/ManageDomainRecordCell.swift:212]: `chains.prefix(3)` is quite specific operation, let's extract into a separate computed var

File: unstoppable-ios-app/domains-manager-ios/Entities/UIMenuDomainAvatarLoader.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+//
+//  Created by Oleg Kuplin on 15.06.2023.
+//
+
+import UIKit
+
+struct UIMenuDomainAvatarLoader {
+    
+    @MainActor
+    static func menuAvatarFor(domain: DomainDisplayInfo,
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/Entities/UIMenuDomainAvatarLoader.swift:13]: I can see a number of this literal `20` -- should be extracted if this is the same in all places

**Review by rommex — APPROVED** [review]
Approved with minor notes

File: unstoppable-ios-app/domains-manager-ios/Entities/UIMenuDomainAvatarLoader.swift (hunk 2 above)
Comment (Oleg-Pecheneg) [reply unstoppable-ios-app/domains-manager-ios/Entities/UIMenuDomainAvatarLoader.swift:13]: I've checked the codebase, after the update there's just one place with number '20', default value for function argument 👍

File: unstoppable-ios-app/domains-manager-ios/Modules/DomainProfile/Cells/ManageDomainRecordCell/ManageDomainRecordCell.swift (hunk 1 above)
Comment (Oleg-Pecheneg) [reply unstoppable-ios-app/domains-manager-ios/Modules/DomainProfile/Cells/ManageDomainRecordCell/ManageDomainRecordCell.swift:212]: Checked the logic. This is the only place (subtitle of ui action row) where we should shrink chain's abbreviation to 3 chars. Don't think it deserves dedicated var in extension that will be visible everywhere. But i will create a dedicated private function in the cell with clear name 👍

**Pull request merged**